services/web/server/tests/integration/01/test_garbage_collection.py

```diff
@@ -81,6 +81,11 @@ async def __delete_all_redis_keys__(redis_settings: RedisSettings):
     # do nothing on teadown
 
 
+@pytest.fixture(scope="session")
```

Comment:
I think at some point we wanted to remove the usage of the session fixtures, since they create issues when debugging. Especially in this situation having a function fixture has virtually no performance impact.

I think @pcrespov also agrees on this.

Reply:
this is a string. therefore I do not really see the problem.
the session fixture is a problem when using the monkeypatch fixture.
Please confirm @pcrespov, thanks.

Reply:
@GitHK @sanderegg in this case (with an unmutable string) i do not see the harm/advantage of it. 
The problems might arise with mutable objects (that if not deep-copied can be modified from test to test) or with mocked environment variables. In those cases, function-session fixtures is highly advisable.